llvm_ir_dataset_utils/analysis_tools/batch_main_body.sh

```diff
@@ -0,0 +1,29 @@
+BATCH=$(($SIZE/$SLURM_ARRAY_TASK_MAX))
+I=$((${SLURM_ARRAY_TASK_ID}*${BATCH}+1+${START}))
+STOP=$(($I+${BATCH}-1))
+if [ $SLURM_ARRAY_TASK_ID -eq $SLURM_ARRAY_TASK_MAX ]; then
+  STOP=$(($I+${SIZE}%$BATCH-1))
+fi
+cd $TMPDIR
+mkdir -p ir_bc_files/ps_$I/${TYPE}
+cd ir_bc_files/ps_$I/${TYPE}
+mkdir -p bc_files instruction_counts perf_stat_files \
+textseg_sizes object_files
+eval tar --extract --file=${STORAGE}${TYPE}/${TYPE}_bc_files.tar \
```

Comment:
If we're going to do this, we need scripts somewhere too to convert the HF version to a collection of tar files.